src/toil/statsAndLogging.py

```diff
@@ -112,9 +112,10 @@ def callback(fileHandle):
                 pass
             else:
                 def logWithFormatting(jobStoreID, jobLogs):
-                    logFormat = '\n%s    ' % jobStoreID
                     logger.debug('Received Toil worker log. Disable debug level '
-                                 'logging to hide this output\n%s', logFormat.join(jobLogs))
+                                 'logging to hide this output')
+                    for line in jobLogs:
+                        logger.debug('%s    %s', jobStoreID, line)
```

Comment:
This will end up with slightly different output. If each line is logged separately they will each be prefixed by logger information (ex: `ip-172-31-5-90.us-west-2.compute.internal 2017-01-09 18:26:49,843 MainThread DEBUG toil.jobStores.aws.jobStore:`) that clutters the output & is unrelated to where/when the logs originated. 

IIRC this is why everything was logged in one statement in the first place, but obviously ugly logs are better than killing the workflow. Ideally we would have both but I couldn't find a reliable/easy way to prevent that log information from showing up for certain messages.